descriptors/ModuleDescriptor-template.json

```diff
@@ -259,7 +260,8 @@
             "consortia.tenants.item.delete"
           ],
           "modulePermissions": [
-            "consortia.consortia-configuration.item.delete"
+            "consortia.consortia-configuration.item.delete",
+            "consortia.identity-provider.item.delete"
```

Comment:
we invoke just service that calls Keycloak, not http call to another Folio service, so module permission is not needed in this case